src/index.js

```diff
@@ -1,14 +1,101 @@
-import _ from 'lodash';
 import './style.css';
+import TasksManager from './tasks_manager';
+import DisplayManager from './display_manager';
 
-function component() {
-  const element = document.createElement('div');
+const selector = (element) => document.querySelector(element);
+const selectorAll = (element) => document.querySelectorAll(element);
+const input = selector('input');
+const manager = new TasksManager();
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'ahmed'], ' ');
-  element.classList.add('hello');
+const updateTask = (event, index, focus = false) => {
+  if (event.key === 'Enter' || focus) {
+    manager.updateTask(
+      index,
+      selector(`.task_${index}`).value,
+      selector(`.task_${index}_checkbox`).checked,
+    );
+    selector('.add_task input').focus();
+  }
+};
 
-  return element;
-}
+const updateStatus = (index) => {
+  manager.updateStatus(index, selector(`.task_${index}_checkbox`).checked);
+  selector(`.task[data-id='${index}'] .input_task`).classList.toggle(
+    'completed',
+  );
+};
 
-document.body.appendChild(component());
+const display = () => {
+  DisplayManager.reset(selector('.tasks'));
+
+  manager
+    .getTasks()
+    .forEach((task) => DisplayManager.displayTask(selector('.tasks'), task));
+
+  selectorAll('.fa-trash').forEach((e) => e.addEventListener('click', () => {
+    // eslint-disable-next-line no-use-before-define
+    deleteTask(e.dataset.id);
+  }));
+
+  selectorAll('.input_task').forEach((e) => e.addEventListener('keyup', (event) => {
+    updateTask(event, e.dataset.id);
+  }));
+
+  selectorAll('.checkbox_task').forEach((e) => e.addEventListener('change', () => {
+    updateStatus(e.dataset.id);
+  }));
```

Comment:
- [ ] Take a look at the image below. At the initial addition of a task, I am able to toggle the `true` and `false` values for **completed** in the localStorage. Now when I reload the page:

  - Firstly, the checkbox is not checked for tasks with `completed: true` values. Kindly fix this.
    **Suggestion:** In your **display_manager** file under `displayTask()`, after creating the input, you could put a conditional and set the attribute `checked` to `true` for checkboxes for tasks with `completed: true` values.
  - Secondly, the `localStorage` only updates for new tasks, not ones that existed before page reload. When I try to change the checkbox for loaded tasks (from the localStorage), the change does not reflect on the `localStorage`. From the image, **`second task`** is checked but value is not updated in the localStorage. Kindly fix this.

  ![ahmed-check](https://user-images.githubusercontent.com/36542986/149435996-a7f1df53-5a23-4feb-b901-dd7159c56fe1.jpg)

Reply:
Hi @TedLivist 
i don't understand this part 
> * Secondly, the `localStorage` only updates for new tasks, not ones that existed before page reload.

Can you explain more please

Reply:
Hey @ahmedtaa. I meant that your change status function does not update the `completed` property of items loaded from the localStorage. Your function only updates the property for items added before page reload.

Reply:
Done 👍 
And thanks for help 💯